### ISSUE TITLE
`ImportError` using Pyjion 1.2.1

### ISSUE THREAD

**Issue by SarthakJariwala** [tonybaloney/Pyjion#465]
Using Pyjion 1.2.1 and Python 3.10.1 (and 3.10.0) on MacOS (12.0.1, Intel), I am getting `ImportError` and `ModuleNotFoundError` : 

`ModuleNotFoundError: No module named 'pyjion._pyjion'`

```
ImportError:
Failed to import the compiled Pyjion module. This normally means something went wrong during pip install
and the binaries weren't compiled. Make sure you update pip before installing to get the right wheel.
If that doesn't work, run pip in verbose mode, or file an issue at https://github.com/tonybaloney/pyjion/.
```

I am using the latest version of pip.

Installing and running Pyjion 1.2.0 worked without any errors. 

Looks like changes between 1.2.0 and 1.2.1 are affecting the binary compilation? Happy to provide more information, if needed.

**Comment by tonybaloney** [comment]
Thanks for the report, the wheel is shipped using the `_pyjion.cpython-310d-darwin.so` library, which is the wrong one. I'll track down what happened.

**Comment by tonybaloney** [comment]
Yanked 1.2.1 for now, roll back to 1.2.0 as a workaround

**Comment by tonybaloney** [comment]
Should be fixed in 1.2.2

**Comment by SarthakJariwala** [comment]
Yup, it's fixed in 1.2.2. Thanks for a very quick turnaround!
